docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -0,0 +1,153 @@
+# Install IPEX-LLM on Windows with Intel NPU
+
+IPEX-LLM provides NPU support for LLM acceleration on Intel Core™ Ultra Processers (Series 2), and offers both Python and C++ API. This guide demonstrates:
```

Comment:
Remove `IPEX-LLM provides NPU support for LLM acceleration on Intel Core™ Ultra Processers (Series 2), and offers both Python and C++ API.`